src/web/app/components/loading-spinner/loading-spinner.component.html

```diff
@@ -1,4 +1,5 @@
 <div class="loading-container">
   <div class="row spinner-border text-primary" role="status"></div>
   <div class="row text-secondary">Loading...</div>
+  <div class="row text-info">{{loadingMessage}}</div>
```

Comment:
Instead of adding another message underneath, I think it would look better to update the original message `Loading...` to `Still loading...`.

Alternatively, as you mentioned, would it be possible to dynamically cycle the loading message (E.g. `Loading.` -> `Loading..` -> `Loading...` -> `Loading.` -> ...). I think that would look cooler and give the user more reassurance :)